vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectLinksComponent.java

```diff
@@ -0,0 +1,31 @@
+package life.qbic.datamanager.views.project.create;
+
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import java.util.List;
+import life.qbic.datamanager.views.components.CardLayout;
+import life.qbic.projectmanagement.domain.finances.offer.Offer;
+
+/**
+ * A component displaying all links of a project
+ */
+public class ProjectLinksComponent extends Composite<CardLayout> {
+
+  private final LinkList<Offer, OfferLinkComponent> offerLinks;
```

Comment:
Why a linked list? I think a grid would be more suitable since we will collect more than just offers in the future

Reply:
It is a link list :) this is a component I implemented